Easydict/App/Localizable.xcstrings

```diff
@@ -1135,18 +1135,18 @@
         }
       }
     },
-    "force_auto_get_selected_text" : {
+    "force_get_selected_text" : {
       "localizations" : {
         "en" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "Force get selected text when auto get selected text failed (Experimental)"
+            "value" : "Allow to use force to get selected text (experimental feature)"
```

Comment:
What about `Enable/Allow forced text selection (experimental feature)`

Reply:
good, it's `Allow forced text selection (note that it may affect clipboard copying)` now.